python/custom_test_files.py

```diff
@@ -0,0 +1,189 @@
+#The following code imports the required packages.
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import shutil as sh
+
+from lar_generator import lar_gen #Imports lar_gen class. 
+lg = lar_gen() #Instantiates lar_gen class as lg. 
+
+class Custom_Test_Files(object):
```

Comment:
What do you think about renaming this class and file to be consistent with the large test files terminology?